lutris/sysoptions.py

```diff
@@ -227,6 +227,19 @@ def get_vk_icd_choices():
         "advanced": True,
         "help": _("Switch to US keyboard QWERTY layout while game is running"),
     },
+    {
+        "option": "nvidia_separate_glshaderdiskcache",
+        "type": "bool",
+        "default": True,
+        "label": _("Separate NVIDIA Shader Cache Location"),
+        "advanced": True,
+        "help": _("Use a dedicated location for a game's shader cache generated by the NVIDIA driver. "
+                  "This bypasses the default shader cache size limitation in the NVIDIA driver and "
+                  "avoids unnecessary cache pruning for games with many shaders. "
+                  "Disable to use the default (shared) shader cache location. If enabled, this will apply "
```

Comment:
"If enabled, this will apply" ?
seems like the sentence ends unfinished

Reply:
The next line continues the sentence with "__GL_SHADER_DISK_CACHE=1" and so on. I just didn't change that line.

I modeled this on the "prime" setting, which explains that it sets environment variables with similar text.

Reply:
Oh I derped out, i see now.